### ISSUE TITLE
Gumfury subdomain takeover

### ISSUE THREAD

**Issue by khaledibnalwalid** [EdOverflow/can-i-take-over-xyz#154]
## Service name
Gumfury

## Proof
https://khaledibnalwalid.wordpress.com/2020/06/25/gemfury-subdomain-takeover/

**Comment by th3r4id** [comment]
Looks like edge case getting an error "Name already exists"

**Comment by Am-MarkhoR-225** [comment]
> Looks like edge case now getting an error "Name already exists" @EdOverflow


> Looks like edge case now getting an error "Name already exists" @EdOverflow

true, i just got the same, but also the page looks differecent then @khaledibnalwalid shared, looks something like http request logging, i tried to claim it but it said "Name Already Exists"

**Comment by ZishanAdThandar** [comment]
### LOOKS LIKE NOT VULNERABLE
I scanned with subzy. Result showed like that. And during takeover it shows "**Name already exists**".

![1](https://user-images.githubusercontent.com/16804210/210379338-bdce9cda-454b-43e1-aa30-05594253fc73.png)
![2](https://user-images.githubusercontent.com/16804210/210379346-4aca689e-77f5-4adf-8a06-7a175a78ed0e.png)

**Comment by bhataasim1** [comment]
I think this is NOT VULNERABLE

**Comment by wadethrillson** [comment]
i added names to my domains but now it says dns validation issue. how to fix that?

**Comment by itsryuku** [comment]
Thu May 23 10:32:27 AM EDT 2024, didn't work for me.

**Comment by khaledibnalwalid** [comment]
I think it has been resolved from the vendor

**Comment by neonn0d** [comment]
I am getting false positive results too

**Comment by RedNotAFatPenquin** [comment]
![image](https://github.com/user-attachments/assets/03dbf9cd-2a22-4a55-9059-15e4c2ac3e1a)
i got this

**Comment by ceylanb** [comment]
Not vulnerable. I tried 10+ subdomains matching the vulnerability conditions, but "Name already exists" returned.

**Comment by hackinwolf** [comment]
Looks like this issue is patched. Before activating the domain, you'll need to verify ownership by adding a CNAME record to your DNS settings.

![Image](https://github.com/user-attachments/assets/0710e5b8-31d0-404f-90c4-b456273bbd13)

**Comment by ghost** [comment]
> Looks like this issue is patched. Before activating the domain, you'll need to verify ownership by adding a CNAME record to your DNS settings.
> 
> ![Image](https://github.com/user-attachments/assets/0710e5b8-31d0-404f-90c4-b456273bbd13)

Bro same thing happened to me what to do next pls any one help me

**Comment by ghost** [comment]
Bro that subdomain added but it show not configured what to do
Is it vulnerable or patched?
